src/qt/ui/modalview.ui

```diff
@@ -95,40 +95,81 @@
       <x>-1</x>
       <y>-1</y>
       <width>401</width>
-      <height>359</height>
+      <height>442</height>
```

Comment:
I noticed that. This section is contains the Qt elements that were changed in this PR. QtCreator made the edit, but I intentionally left it as I assumed it did this because of the extra line edit that was added. I just tested it with the previous `359` and didn't notice a difference.